### PR TITLE
[15.0][MIG] project_operating_unit: migration to 15.0

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

File: project_operating_unit/models/project_task.py [hunk 1]
```diff
@@ -11,7 +11,4 @@ class ProjectTask(models.Model):
         comodel_name="operating.unit",
         related="project_id.operating_unit_id",
         string="Operating Unit",
-        default=lambda self: self.env["res.users"].operating_unit_default_get(
```
Comment (alan196) [inline project_operating_unit/models/project_task.py]: Why do you remove the default?

**Review by alan196 — CHANGES_REQUESTED** [review]
Only one comment

File: project_operating_unit/models/project_task.py (hunk 1 above)
Comment (huguesdk) [reply project_operating_unit/models/project_task.py]: thanks for the review!

with the default, odoo gives the following warning:
```
odoo.fields: Redundant default on project.task.operating_unit_id
```
indeed, it’s a related field and the default value is the same on the original field.

Comment (alan196) [reply project_operating_unit/models/project_task.py]: Sorry, I didn't notice it was a related field.

**Review by alan196 — APPROVED** [review]
👍 Technical review

**Comment by alan196** [comment]
@EduardoJarsa @hectorHerreraTeigeiro could you help me to review this PR?

**Review by EduardoJarsa — COMMENTED** [review]
functional review everything looks working

**Review by EduardoJarsa — APPROVED** [review]
Functional review, everything looks working

File: project_operating_unit/models/project_task.py (hunk 1 above)
Comment (hectorHerreraTeigeiro) [reply project_operating_unit/models/project_task.py]: Looks complete for me

**Pull request merged**